### DISCUSSION TITLE
INFO: Cannot ensure issue because issues are disabled in this repository

### DISCUSSION THREAD

**Discussion by xiabee** [renovatebot/renovate#24172] [Request Help]
### How are you running Renovate?

Placeholder value, please select the correct response from the dropdown

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

I installed renovate in my forked repository [tidb](https://github.com/xiabee/tidb), somtimes it worked but sometimes not.

The interactive mode has been set up before, but the Pr has not been received. This time I received the pr of the configuration file, run it again after merging, and can't get the Pr of the security repair. Run log prompted "Cannot ensure issue because issues are disabled in this repository".

Is this because my warehouse detects too many files and exceeds the usage limit of the SaaS service, causing the detection to fail?

Sometimes it works and sometimes it doesn't. This bug is not easy to reproduce.

### Relevant debug logs

<details><summary>Logs</summary>

```
...
INFO: Cannot ensure issue because issues are disabled in this repository
INFO: Cannot ensure issue because issues are disabled in this repository
INFO: Cannot ensure issue because issues are disabled in this repository
INFO: Repository finished
{
  "cloned": true
  "durationMs": 37270
}
...
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Your repository doesn't have issues enabled

**Reply by xiabee** [reply]
What do issues here refer to? My settings are the same as those of other repositories (for example, [tidb-dashboard](https://github.com/xiabee/tidb-dashboard/pulls)), but only the [tidb](https://github.com/xiabee/tidb) repository cannot be detected.

**Reply by rarkins** [reply]
GitHub Issues in that repository
